composer.json

```diff
@@ -18,7 +18,8 @@
 
         "smarty/smarty": "^3.1",
         "geshi/geshi": "^1.0.9",
-        "setasign/fpdf": "^1.8"
+        "setasign/fpdf": "^1.8",
+        "symfony/debug": "^3.4"
```

Comment:
`require-dev` are dependencies that are not necessary to run LANSuite.
If an enduser enabled the debugmode (when the user downloads the released package), it is not there and a fatal error will be thrown.

In this case here, it don't add overhead.

Reply:
That is a weak argument, isn't it?
If you want to have a stack trace, then to fix a code / DB issue, thus you are developing.
But OK, mostly because we lack a better way to get end users to provide us with good error reporting